modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -48,79 +49,151 @@ class FeatureFloodCount :
 
   virtual void initialize();
   virtual void execute();
-//  virtual void threadJoin(const UserObject & y);
   virtual void finalize();
   virtual Real getValue();
 
-  // Retrieve field information
-  virtual Real getNodalValue(dof_id_type node_id, unsigned int var_idx=0, bool show_var_coloring=false) const;
-  virtual Real getElementalValue(dof_id_type element_id) const;
+  enum FIELD_TYPE
+  {
+  UNIQUE_REGION,
```

Comment:
That's emacs fault. I guess I really need to add more doco too before this
is merged

On Wed, Apr 6, 2016 at 5:52 PM Daniel Schwen <notifications@github.com>
wrote:

> In modules/phase_field/include/postprocessors/FeatureFloodCount.h
> <https://github.com/idaholab/moose/pull/6713#discussion_r58801320>:
>
> >    virtual void finalize();
> >    virtual Real getValue();
> >
> > -  // Retrieve field information
> > -  virtual Real getNodalValue(dof_id_type node_id, unsigned int var_idx=0, bool show_var_coloring=false) const;
> > -  virtual Real getElementalValue(dof_id_type element_id) const;
> > +  enum FIELD_TYPE
> > +  {
> > +  UNIQUE_REGION,
>
> no indent?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6713/files/84ad9d55b4e0aa2a2509c6255c41f80fb270d5c6#r58801320>
>